mlflow/gateway/constants.py

```diff
@@ -2,6 +2,7 @@
 MLFLOW_GATEWAY_CRUD_ROUTE_BASE = "/api/2.0/gateway/routes/"
 MLFLOW_GATEWAY_ROUTE_BASE = "/gateway/"
 MLFLOW_QUERY_SUFFIX = "/invocations"
+MLFLOW_GATEWAY_NUM_ROUTES_PER_SEARCH_RESULTS_PAGE = 3000
```

Comment:
/ai The following constant is used by `mlflow.search_routes` (which supports pagiation) and represents the page size. Suggest 5 names for this constant.

Reply:
I'd use `MLFLOW_GATEWAY_SEARCH_ROUTES_PAGE_SIZE`.

Reply:
Great suggestion! Changed!